### ISSUE TITLE
README.md request

### ISSUE THREAD

**Issue by Xyndra** [FoundryMC/Alembic#19]
Please add a README.md so that it is obvious what this repository is about

**Comment by meeprdib** [comment]
https://github.com/FoundryMC/Alembic/wiki

**Comment by IDoTheHax** [comment]
That's a wiki not really a readme

**Comment by Xyndra** [comment]
A README.md is there to be visible on first view. The wiki is not something that immediately pops up

**Comment by meeprdib** [comment]
True, although the wiki *is* pretty explanatory on what the mod is.

**Comment by IDoTheHax** [comment]
It can be a summary and have links to the discord for help (for example)

**Comment by Xyndra** [comment]
> True, although the wiki _is_ pretty explanatory on what the mod is.

I never questioned that
